docs/application/dotnet/guides/app-management/application-launcher.md

```diff
@@ -0,0 +1,301 @@
+        {
+            Size2D = new Size2D(size.Height - 40, size.Height - 40),
+            ResourceUrl = path,
+        };
+
+        this.Layout = new LinearLayout()
+        {
+            LinearAlignment = LinearLayout.Alignment.CenterHorizontal,
+            LinearOrientation = LinearLayout.Orientation.Vertical
+        };
+
+        this.Add(Icon);
+        this.Add(label);
+
+        this.TouchEvent += OnTouchEvent;
+    }
+    ```
+
+    **Figure: Application Icons**
+
+    ![Application Icons](./media/application_launcher_icons.png)
+
+6. `OnTouchEvent` reads state from the `TouchEventArgs`. If touch is in `PointStateType.Down` state, the application icon is resized. Otherwise, it returns to normal size and predefined event is invoked with the proper `AppId`:
+
+    ```csharp
+    public bool OnTouchEvent(object sender, TouchEventArgs args)
+    {
+        var state = args.Touch.GetState(0);
+
+        switch (state)
+        {
+            case PointStateType.Down:
+                Icon.Size2D = new Size2D((int)(OriginSize.Width * 1.1f), (int)(OriginSize.Height * 1.1f));
+                break;
+            case PointStateType.Finished:
+                Icon.Size2D = OriginSize;
+                ApplicationIconClicked.Invoke(this, new ApplicationIconClickedEventArgs(this.AppId));
+                break;
+            default:
+                break;
+        }
+
+        return false;
+    }
+    ```
+
+7. The `Program` class derived from NUIApplication handles all necessary system events ([Application Lifecycle](../applications/uiapplication/application_lifecycle.md)). The `AppLauncher` is created in `Initialize()` method and used in an icon touch handler:
+
+    ```csharp
+    class Program : NUIApplication
+    {
+        private AppControl AppLauncher;
+
+        //...
+    }
+    ```
+
+8. The `OnCreate()` method calls `Initialize()` function before the main application loop starts:
+
+    ```csharp
+    protected override void OnCreate()
+    {
+        base.OnCreate();
+        Initialize();
+    }
+    ```
+
+9. The `Initialize()` method set up key listener, `AppControl` object, and the application background:
+
+    ```csharp
+    void Initialize()
+    {
+        var appWindow = Window.Instance;
+        appWindow.KeyEvent += OnKeyEvent;
+
+        AppLauncher = new AppControl();
+        ImageView background = new ImageView(DirectoryInfo.Resource + "/images/bg.png");
+
+        //Setting the background parameters so that it occupies the entire application window
+        background.Size2D = new Size2D(appWindow.Size.Width, appWindow.Size.Height);
+        background.Position2D = new Position2D(0, 0);
+        appWindow.GetDefaultLayer().Add(background);
+    }
+    ```
+
+10. In the next step, grid component for application icons is created. Grid spacing and columns number are defined in the [GridLayout](/application/dotnet/guides/nui/grid-layout.md) object. The `appGrid` component width and height is set to fill its parent:
```

Comment:
\[GridLayout](/application/dotnet/guides/nui/grid-layout.md)

Would you please check if this URL is valid with the master branch?

dotnet/guides/nui/grid-layout.md is moved to dotnet/guides/user-interface/nui/grid-layout.md by https://github.com/Samsung/tizen-docs/pull/1511.

Reply:
My worrying point was also fixed.